functions/pupil-login/v1.spec.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+/* global describe expect it spyOn */
+
+const sqlService = require('less-tedious')
+const v1 = require('./v1')
+
+describe('v1-process', () => {
+  it('sqlService.modify', async () => {
```

Comment:
@jon-shipley can you please provide a better description for that test?
Doesn't seem very readable to me.

Reply:
Me Neither ☹️ I'll fix it.